tests/tt_eager/python_api_testing/unit_testing/test_optimized_conv_v2.py

```diff
@@ -48,6 +48,7 @@
         # Small conv
         (1, 32, 32, 16, 16, 3, 3, 2, 2, 1, 1, True),
         # (1, 32, 32, 16, 16, 3, 3, 2, 2, 1, 1, False), # Asserts #5323
+        (2, 256, 128, 3, 3, 3, 3, 1, 1, 0, 0, True),
```

Comment:
Please move this test to `test_conv.py. We want to deprecate this test file.

Reply:
@tt-nshanker, done! Moved this test to the ttnn version